tools/flowc/tests/cpp3/test_compile_time.flow

```diff
@@ -0,0 +1,231 @@
+import text/binary;
```

Comment:
Is this test only relevant for Cpp, or is it a more general test case?

Reply:
Both, c++ and java backends pass tests with optimization.